en/components/general/wpf_to_angular_guide/angular_events.md

```diff
@@ -1,16 +1,14 @@
 ---
-title: Angular Events
-_description: Learn how to respond to events in Angular and implement your own events.
-_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, Events, Event Binding
+title: Handling Events in Angular | IgniteUI for Angular | Infragistics
```

Comment:
`angular events` would be a better choice for a primary keyword

Reply:
Addressed.